eq-author/src/App/page/Preview/CommentsPanel/index.js

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react";
+import { withRouter } from "react-router";
+import { get } from "lodash";
+import moment from "moment";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import Loading from "components/Loading";
+import Error from "components/Error";
+import CustomPropTypes from "custom-prop-types";
+// import PropTypes from "prop-types";
+
+import COMMENT_QUERY from "./commentsQuery.graphql";
+import COMMENT_ADD from "./createNewComment.graphql";
+
+import styled from "styled-components";
+import ScrollPane from "components/ScrollPane";
+import { radius, colors } from "constants/theme";
+
+import TextArea from "components/Forms/TextArea";
+import { Field, Label } from "components/Forms";
+import IconDelete from "./icon-delete.svg";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+
+const CommentsPane = styled.div`
+  background: ${colors.white};
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  font-size: 1em;
+`;
+
+const CommentSection = styled.div`
+  padding: 0.7em 1em;
+  border-bottom: 1px solid ${colors.lightMediumGrey};
+`;
+
+const ShowComment = styled.div`
+  align-items: left;
+  padding: 0.5em 1em;
+  margin-bottom: 5px;
+  border: 1px solid ${colors.lightGrey};
+  background-color: ${colors.lighterGrey};
+`;
+
+const CommentHeaderContainer = styled(Field)`
+  display: flex;
+  justify-content: flex-start;
+  -webkit-align-items: start;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 8px;
+`;
+
+const AvatarWrapper = styled.div`
+  flex-grow: 0;
+  margin-right: 8px;
+`;
+
+const AvatarOuter = styled.div`
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+  background-color: ${colors.primary};
+  text-align: center;
+`;
+
+const AvatarInner = styled.div`
+  color: ${colors.white};
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 0;
+  padding: 25% 0;
+`;
+
+const NameWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  -webkit-flex-direction: column;
+  justify-content: space-between;
+  align-items: start;
+  flex-grow: 2;
+`;
+
+const FlexLabel = styled.div`
+  font-size: 1.1em;
+  align-items: center;
+  height: 22px;
+`;
+
+const DateField = styled("span")`
+  font-weight: normal;
+  font-size: 0.8em;
+  color: ${colors.grey};
+`;
+
+const DeleteButtonIcon = styled.button`
```

Comment:
We already have a component for this sorta thing, think its called an IconButton.

Reply:
removed both the delete and reply buttons until those functions are added